### DISCUSSION TITLE
Commiting into main branch

### DISCUSSION THREAD

**Discussion by bp26** [codesandbox/codesandbox-client#7398] [Q&A] [answered]
How do I actually work with my main branch of imported github repository? It shows as protected. It was possible in the previous interface to link sandbox to repository and commit into the main branch.

**Comment by tristandubbeld** [comment]
Hey @bp26, great question! We were running into some technical limitations in the past that made us treat the default branch as protected. However, those limitations are gone and we're actively working to fix this! In the future we hope to make this toggle-able from the editor too. For now you still have to branch out to make updates but we hope to be able to deliver this feature soon.

**Comment by Ekwuno** [comment, marked as answer]
Hey @bp26, we now have toggling protection for branches. Here is a link to the docs https://codesandbox.io/docs/learn/environment/branch-protection
